iib/workers/tasks/build.py

```diff
@@ -763,6 +767,10 @@ def handle_add_request(
             overwrite_from_index_token,
         )
 
+        _add_ocp_label_to_index(
```

Comment:
We might want to add this for `rm` request as well right? https://github.com/release-engineering/iib/pull/116/files#diff-055f6f8c453137f787488a4d047253a1R806

Reply:
added